vpr/src/place/net_cost_handler.h

```diff
@@ -169,6 +169,21 @@ class NetCostHandler {
     /* [0...num_affected_nets] -> net_id of the affected nets */
     std::vector<ClusterNetId> ts_nets_to_update_;
 
+    // [0..cluster_ctx.clb_nlist.nets().size()-1]. Store the number of blocks on each of a net's bounding box (to allow efficient updates)
```

Comment:
I think it would be useful to add a top-level comment here explaining that the following variables store the bounding box coordinates, etc., after a move is committed (to clarify how they differ from the ts_ variables).